ssc/Pos/Ssc/Extra/Logic.hs

```diff
@@ -235,9 +235,6 @@ sscApplyBlocksFinish
     => SscGlobalState ssc -> m [SomeBatchOp]
 sscApplyBlocksFinish gs = do
     sscRunGlobalUpdate (put gs)
-    inAssertMode $
-        logDebug $
-        sformat ("After applying blocks SSC global state is:\n"%build) gs
```

Comment:
I asked about it and it turned out there is no way to do it with current `log-warper`.
But actually I find it suspicious. If I run demo locally (in master), it's not a huge text. So I guess it's a huge text only if you run benchmarks (and have many nodes). But if you run benchmarks, you shouldn't use assert mode. You can disable it by turning off `asserts` flag in `cardano-sl-core`.